Gems/GradientSignal/Code/Source/UI/GradientPreviewDataWidget.cpp

```diff
@@ -110,13 +109,11 @@ namespace GradientSignal
         layout->setContentsMargins(QMargins());
         layout->setAlignment(Qt::AlignHCenter);
 
-        m_preview = new GradientPreviewWidget(this);
+        m_preview = new GradientPreviewWidget(true, this);
```

Comment:
Suggestion: Consider using something like:
```
constexpr bool enablePopout = true;
m_preview = new GradientPreviewWidget(enablePopout, this);
```
It provides more readability on what "true" means in this context.

Reply:
I like that, thanks!